### DISCUSSION TITLE
The colors of buttons still black

### DISCUSSION THREAD

**Discussion by ElizabethB7** [saadeghi/daisyui#3087] [Q&A]
Hey, I'm trying bulid webtise with daisyui. 
I want my buttons will be in colors and I still not seeing any changes in button colors.
//App.jsx
import "./App.css";
import React from "react";

const MyComponent = () => (
  <div>
    <button class="btn btn-primary">One</button>
    <button class="btn btn-secondary">Two</button>
    <button class="btn btn-accent btn-outline">Three</button>
  </div>
);

export default MyComponent;

//tailwind.config.js
module.exports = {
  content: ["./src/**/*.{js,jsx,ts,tsx}"],
  theme: {
    extend: {},
  },
  plugins: [require("daisyui")],
  daisyui: {
    themes: ["light", "dark", "cupcake"], // add themes as needed
  },
  safelist: [
    "btn",
    "btn-primary",
    "btn-secondary",
    "btn-accent",
    "btn-info",
    "btn-success",
    "btn-warning",
    "btn-error",
  ],
};

//App.css
@tailwind base;
@tailwind components;
@tailwind utilities;
#root {
  max-width: 1280px;
  margin: 0 auto;
  padding: 2rem;
  text-align: center;
}

**Comment by saadeghi** [comment]
Please share a minimal reproduction link on GitHub or Tailwind Play or Stackblitz.
